NAMESPACE

```diff
@@ -66,7 +66,15 @@ export(local_image)
 export(matches)
 export(md)
 export(one_of)
+export(opt_add_row_striping)
+export(opt_add_table_outline)
+export(opt_align_table_header)
+export(opt_all_caps)
 export(opt_footnote_marks)
+export(opt_fully_lined)
+export(opt_lineless)
+export(opt_remove_table_outline)
+export(opt_remove_top_bottom_borders)
```

Comment:
Change to:

* `opt_row_striping(TRUE/FALSE)`
* `opt_table_outline(TRUE/FALSE)`
* `opt_align_table_header` (unchanged)
* `opt_all_caps(TRUE/FALSE)`
* `opt_footnote_marks` (unchanged, but add validation to `marks` and no default)
* `opt_table_lines("all"/"none"/"default")`?
* (drop `opt_remove_top_bottom_borders`)

Reply:
All changed now!